[esx]/esx_skin/client/main.lua

```diff
@@ -1,6 +1,13 @@
 local lastSkin, cam, isCameraActive
 local firstSpawn, zoomOffset, camOffset, heading, skinLoaded = true, 0.0, 0.0, 90.0, false
 
+RegisterNetEvent('esx:playerLoaded')
+AddEventHandler('esx:playerLoaded', function(xPlayer, isNew, skin)
+    if ESX.IsPlayerLoaded() then
```

Comment:
@SirLamaGott Remove please this if statement, i think don't need check player is loaded if we call event in esx:playerLoaded event. 

I tested and i get false ESX.IsPlayerLoaded() all time.

Reply:
I removed it.